### ISSUE TITLE
IE options toggle images/css/javascript/activex don't work

### ISSUE THREAD

**Issue by stevefaulkner** [ThePacielloGroup/WebAccessibilityToolbar#89]
the toggle images/css/javascript/activex in IE menu don't work  using IE11, windows 8.1 using isntalled version 2015-02-04

**Comment by menovak** [comment]
hi Steve

These 4 functions do not work on 32 bit version either (Windows 7/IE11) and
I can't recall when they did work, also at a quick glance i'm not exactly
sure what they call/invoke?

toggleimg=Toggleimg
ToggleJS=Togglejs
ToggleAX=Toggleax
ToggleCSS=Togglecss

Mark

**Comment by jun325** [comment]
The "WAT_IESet.exe" is necessary for those functions.
I don't understand why it was deleted.

**Comment by stevefaulkner** [comment]
@jun325 thanks, @menovak @matatk can the WAT_IESet.exe be added back and a new release made ASAP?

**Comment by matatk** [comment]
The `WAT_IESet.exe` file will still be in the history for the repository, so should be easy to restore (in theory).  I have the following questions about restoring it, to make sure it's done correctly:

 1. Where should it go? (Assuming the same place it was before.)
 2. Are there separate 32- and 64-bit versions?
 3. Does anything need to be done at install time for it, or is it just called when required?

Unfortunately I am unlikely to be able to work on this until later this week, sorry. Will get to it as soon as I can, of course.

**Comment by matatk** [comment]
Further to my last comment, I've realised that we presently have two different ways of disabling/removing CSS and images, as the following WAT options still work: 

 * CSS > Disable CSS
 * Images > Remove Images

Is there a difference between these and the ones on the 'IE' menu (that's WAT > IE > *)?  Do we need both different ways to do the same thing?  I suspect that 'Images > Remove Images' may be different to the IE-based approach, because it leaves a note where the image was, but as the IE-based one isn't working at the moment I can't tell for sure.

@stevefaulkner @menovak: what are your thoughts on this?

**Comment by stevefaulkner** [comment]
@matatk the IE options actually disable stuff in the browser (by changing browser settings) and do so until they are enabled again. The 2 features you mention above are 'simulations'

**Comment by matatk** [comment]
@stevefaulkner: got it -- though are both really needed? I think you are saying they are -- though I feel we should also improve the docs (and possibly even rename the menu options in WAT) to reflect this. I'll file an issue on that, too, and assign myself...

**Comment by menovak** [comment]
Please leave the WAT version in the CSS menu as is... I've found it more
reliable and makes more sense grouped where it is.

**Comment by stevefaulkner** [comment]
where are we at with this?

**Comment by matatk** [comment]
I will work on this as soon as I'm able. If @jun325 could send answers to [the three questions above](https://github.com/ThePacielloGroup/WebAccessibilityToolbar/issues/89#issuecomment-98986682) that'd help (particularly questions 2 and 3), then I should be able to just go in and fix it.

**Comment by jun325** [comment]
>  1. Where should it go? (Assuming the same place it was before.)

It's the same place.

>  2. Are there separate 32- and 64-bit versions?

Yes.

>  3. Does anything need to be done at install time for it, or is it just called when required?

It's just called.

**Comment by matatk** [comment]
Thanks @jun325; I'm on it... I think I deleted them originally because I misread something in a previous issue; sorry!

**Comment by matatk** [comment]
I have just uploaded [a new test release](https://github.com/ThePacielloGroup/WebAccessibilityToolbar/releases/tag/2015-05-20) with the IE settings stuff restored.  However, I noticed that, when I open the `IE` > `Internet Explorer Accessibility Options` dialog, then try clicking on `Captions` button in the bottom-left, nothing happens.  I believe [I restored everything that was needed](https://github.com/ThePacielloGroup/WebAccessibilityToolbar/commit/538a8002264792df17ea71420387126c499027f0), so am wondering if this represents a separate, but related, bug?

**Note:** Until we have resolved whether the captions thing is a separate bug, I've marked the release I've just made as non-production-ready, and I am not merging in my changes to `master` just yet.
